metaflow/runtime.py

```diff
@@ -636,20 +643,11 @@ def persist(self, flow):
         self._ds.persist(flow)
         self._ds.done()
 
-    def save_logs(self, logtype, logs):
-        location = self._ds.save_log(logtype, logs)
-        datum = [MetaDatum(field='log_location_%s' % logtype,
-                           value=json.dumps({
-                               'ds_type': self._ds.TYPE,
-                               'location': location,
-                               'attempt': self.retries}),
-                           type='log_path',
-                           tags=[])]
-        self.metadata.register_metadata(self.run_id,
-                                        self.step,
-                                        self.task_id,
-                                        datum)
-        return location
+    def save_logs(self, stdout_buffer, stderr_buffer):
```

Comment:
As a comment here: we should probably not remove the metadata information. For the location, we can do like we do for artifacts in the convergence branch which stores the location as ':root:datastore_root'. This does not store the URL (because there would be many in part) but does allow the client to get an instance of the datastore used to stave the logs (and therefore retrieve the logs).

Reply:
Good point. Will do!

Reply:
Can you point me to your change in the convergence branch? I don't think `save_logs` is the place to store the `datastore_root` in the metadata.

Reply:
I just pushed stuff in #479, I took your (offline) suggestion to put the ds-root and ds-type as a task level metadata.

Reply:
I have added those elements in this PR as well.